lib/elixir/lib/exception.ex

```diff
@@ -1257,6 +1278,27 @@ defmodule UndefinedFunctionError do
 end
 
 defmodule FunctionClauseError do
+  @moduledoc """
+  An exception raised when a function invocation doesn't match any defined clause.
```

Comment:
```suggestion
  An exception raised when a function call doesn't match any defined clause.
```

The wording feels slightly simpler (also seems more commonly used based on a quick grep)